jenkinsfiles/kubernetes-upstream.Jenkinsfile

```diff
@@ -1,132 +0,0 @@
-                sh 'env'
-                checkout scm
-                sh 'mkdir -p ${PROJ_PATH}'
-                sh 'ls -A | grep -v src | xargs mv -t ${PROJ_PATH}'
-                sh '/usr/local/bin/cleanup || true'
-            }
-        }
-        stage('Set programmatic env vars') {
-            steps {
-                script {
-                    env.IMAGE_REGISTRY = sh script: 'echo -n ${JobImageRegistry:-quay.io/cilium}', returnStdout: true
-
-                    if (env.ghprbActualCommit?.trim()) {
-                        env.DOCKER_TAG = env.ghprbActualCommit
-                    } else {
-                        env.DOCKER_TAG = env.GIT_COMMIT
-                    }
-                    if (env.run_with_race_detection?.trim()) {
-                        env.DOCKER_TAG = env.DOCKER_TAG + "-race"
-                        env.RACE = 1
-                        env.LOCKDEBUG = 1
-                        env.BASE_IMAGE = "quay.io/cilium/cilium-runtime:65ff7285a062ec058e5f9b7b1d18d79fcbf8dc4b@sha256:ad919992216ab753a8bcd5a1ba251178058df50fcbb54604445d0804274c8eb0"
-                    }
-                }
-            }
-        }
-        stage('Preload vagrant boxes'){
-            steps {
-                sh '/usr/local/bin/add_vagrant_box ${WORKSPACE}/${PROJ_PATH}/vagrant_box_defaults.rb'
-            }
-        }
-        stage('Boot VMs'){
-            options {
-                timeout(time: 70, unit: 'MINUTES')
-            }
-
-            steps {
-                retry(3){
-                    sh 'cd ${TESTDIR}; vagrant destroy k8s1-${K8S_VERSION} --force'
-                    sh 'cd ${TESTDIR}; vagrant destroy k8s2-${K8S_VERSION} --force'
-                    sh 'cd ${TESTDIR}; CILIUM_REGISTRY=quay.io timeout 20m vagrant up k8s1-${K8S_VERSION} k8s2-${K8S_VERSION}'
-                }
-            }
-        }
-
-        stage('BDD-tests'){
-            options {
-                timeout(time: 150, unit: 'MINUTES')
-            }
-
-            environment {
-                POLL_TIMEOUT_SECONDS=300
-            }
-
-            steps {
-                sh 'cd ${TESTDIR}; vagrant ssh k8s1-${K8S_VERSION} -c "cd /home/vagrant/go/${PROJ_PATH}; sudo ./test/kubernetes-test.sh ${IMAGE_REGISTRY} ${DOCKER_TAG}"'
-            }
-        }
-
-        stage('Netperf tests'){
-
-            when {
-                environment name: 'GIT_BRANCH', value: 'origin/main'
-            }
-
-            options {
-                timeout(time: 300, unit: 'MINUTES')
-            }
-
-            environment {
-                PROMETHEUS_URL="https://metrics.cilium.io/metrics/job/upstream_job"
-                PROMETHEUS=credentials("metrics")
-            }
-
-            steps {
-                sh '''
-                    cd ${TESTDIR}; vagrant ssh k8s1-${K8S_VERSION} -c "
-                        cd /home/vagrant/go/${PROJ_PATH}/test;
-                        ./kubernetes-netperftest.sh '$PROMETHEUS_URL' '$PROMETHEUS_USR' '$PROMETHEUS_PSW'"
```

Comment:
I looks like `test/kubernetes-netperftest.sh` (and in turn `contrib/scripts/netperf_reporter.py`) are now unused, so these scripts could probably be removed as well.

Reply:
good call!